pkg/internal/config/config_test.go

```diff
@@ -59,18 +59,18 @@ func TestNewShouldApplyDefaultsWhenOptionalFieldsAreMissing(t *testing.T) {
 	actualConfig, err := config.New(baseConfig)
 	assert.NoError(t, err)
 
-	trafficGeneratorMacAddressDefault, _ := net.ParseMAC(config.TrafficGeneratorMacAddressDefault)
+	trafficGeneratorEastMacAddressDefault, _ := net.ParseMAC(config.TrafficGeneratorEastMacAddressDefault)
 	dpdkMacAddressDefault, _ := net.ParseMAC(config.DPDKMacAddressDefault)
 	expectedConfig := config.Config{
 		PodName:                         testPodName,
 		PodUID:                          testPodUID,
 		NUMASocket:                      numaSocket,
 		NetworkAttachmentDefinitionName: networkAttachmentDefinitionName,
 		TrafficGeneratorPacketsPerSecondInMillions: config.TrafficGeneratorPacketsPerSecondInMillionsDefault,
-		PortBandwidthGB:            config.PortBandwidthGBDefault,
-		TrafficGeneratorMacAddress: trafficGeneratorMacAddressDefault,
-		DPDKMacAddress:             dpdkMacAddressDefault,
-		TestDuration:               config.TestDurationDefault,
+		PortBandwidthGB:                config.PortBandwidthGBDefault,
+		TrafficGeneratorEastMacAddress: trafficGeneratorMacAddressDefault,
```

Comment:
Shouldn't `trafficGeneratorMacAddressDefault` be changed to `trafficGeneratorEastMacAddressDefault`?

Reply:
I fixed it in the wrong commit, fixed it here now.